addons/project/tests/test_res_config_settings.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.tests import tagged, Form, TransactionCase
+
+
+@tagged('post_install', '-at_install')
+class TestResConfigSettings(TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.analytic_plan_1 = cls.env['account.analytic.plan'].create({
+            'name': 'Plan 1',
+            'default_applicability': 'optional',
+            'company_id': False,
+        })
+        cls.analytic_plan_2 = cls.env['account.analytic.plan'].create({
+            'name': 'Plan 2',
+            'default_applicability': 'optional',
+            'company_id': False,
+        })
+        cls.company_1 = cls.env['res.company'].create({
+            'name': 'Test Company 1',
+        })
+        cls.company_2 = cls.env['res.company'].create({
+            'name': 'Test Company 2',
+        })
```

Comment:
we could do the create in batch. 🙂 

```suggestion
        cls.company_1, cls.company_2 = cls.env['res.company'].create([
            {'name': 'Test Company 1'}, 
            {'name': 'Test Company 2'},
        ])
```